### ISSUE TITLE
error when installing mapbox-gl

### ISSUE THREAD

**Issue by THartsock02** [mapbox/mapbox-gl-js#4957]
**mapbox-gl-js version**:
when trying to install the latest version of mapbox i get an error

Failed at the mapbox-gl@0.38.0 postinstall script 'cd src/style-spec && yarn || npm install'

I'm on version 6.11.0 for node and 3.10.10 for npm

**Comment by jfirebaugh** [comment]
Can you post the full output of the command please?

**Comment by andrewharvey** [comment]
@jfirebaugh I don't know if it's the same issue as @THartsock02, but it sounds similar due to an error at the postinstall step.

```
mkdir myproject
yarn init
yarn add https://github.com/mapbox/mapbox-gl-js.git
```

```
error "/tmp/myproject/src/style-spec" doesn't exist.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

The `cd src/style-spec` in postinstall is worked in myproject not within mapbox-gl.

I'm installing from github rather than npm as I need to install master.

**Comment by THartsock02** [comment]
Heres the output

> mapbox-gl@0.38.0 postinstall C:\Users\thartsoc\Development\Node\CircleK\node_modules\mapbox-gl
> cd src/style-spec && yarn || npm install

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

npm ERR! file C:\WINDOWS\system32\cmd.exe;C:\Program Files (x86)\Git\bin;C:\Program Files (x86)\Git\cmd
npm ERR! path C:\WINDOWS\system32\cmd.exe;C:\Program Files (x86)\Git\bin;C:\Program Files (x86)\Git\cmd
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn C:\WINDOWS\system32\cmd.exe;C:\Program Files (x86)\Git\bin;C:\Program Files (x86)\Git\cmd
npm ERR! mapbox-gl@0.38.0 postinstall: `cd src/style-spec && yarn || npm install`
npm ERR! spawn C:\WINDOWS\system32\cmd.exe;C:\Program Files (x86)\Git\bin;C:\Program Files (x86)\Git\cmd ENOENT
npm ERR!
npm ERR! Failed at the mapbox-gl@0.38.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\thartsoc\AppData\Roaming\npm-cache\_logs\2017-07-10T14_38_54_239Z-debug.log

**Comment by jfirebaugh** [comment]
Thanks! This was fixed in #4886 for the next release.
